src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -179,9 +180,7 @@ object ScalafixPlugin extends AutoPlugin {
   override lazy val globalSettings: Seq[Def.Setting[_]] = Seq(
     scalafixConfig := None, // let scalafix-cli try to infer $CWD/.scalafix.conf
     scalafixOnCompile := false,
-    scalafixResolvers := Seq(
-      Repository.ivy2Local(),
-      Repository.central(),
+    scalafixResolvers := Repository.defaults().asScala.toSeq ++ Seq(
```

Comment:
Should this only happen if `useCoursier := false` (from the [SBT docs](https://www.scala-sbt.org/1.x/docs/sbt-1.3-Release-Notes.html#Library+management+with+Coursier)).

Maybe it makes more sense to just to tap into [SBT's default resolvers](https://www.scala-sbt.org/1.x/docs/Resolvers.html)?

Reply:
> Should this only happen if useCoursier := false (from the SBT docs).

As of now, as you mention, `scalafix` is bypassing the `sbt` resolvers, so the sbt setting `useCoursier := false` doesn't apply to setting the `scalafixResolvers`. This may come into play if your next suggestion is implemented, though?

> Maybe it makes more sense to just to tap into SBT's default resolvers?

For the particular problem I'm trying to solve at Rally, yes, this would work without needing an env variable override. Assuming all scala projects are using the `rally-sbt-plugin` (private to my company), which puts https://<customRepositoryHost>/ivy-libs-snapshot in `globalSettings`, and our private scalafix rules are published to the same https://<customRepositoryHost>/ivy-libs-snapshot, then using SBT's default resolvers for each repo would work for a tool like `scala-steward`.

Since `scala-steward` adds `sbt-scalafix` to repos on the fly, I could see a scenario where the repos' configured SBT resolvers are not the same as the custom resolver where the scalafix rules are hosted, in which case the env variable would still be desired to add additional custom resolvers.